signal/events_shhext.go

```diff
@@ -218,18 +220,21 @@ func SendNewMessages(obj json.Marshaler) {
 	send(EventNewMessages, obj)
 }
 
-func SendMailserverAvailable(nodeAddress, id string) {
-	send(EventMailserverAvailable, MailserverSignal{
-		Address: nodeAddress,
-		ID:      id,
-	})
+func sendMailserverSignal(ms *mailservers.Mailserver, event string) {
+	msSignal := MailserverSignal{}
+	if ms != nil {
```

Comment:
Does it even make sense to send a signal when `ms == nil`?

Reply:
Good catch. I forgot to change
```
func SendMailserverNotWorking() {
	send(EventMailserverNotWorking, MailserverSignal{})
}
```
to
```
func SendMailserverNotWorking() {
	send(EventMailserverNotWorking, nil)
}
```